pkg/api/scylla/v1/types_cluster.go

```diff
@@ -331,6 +331,14 @@ type SchedulerTaskSpec struct {
 	// +kubebuilder:default:=3
 	// +optional
 	NumRetries *int64 `json:"numRetries,omitempty"`
+
+	// cron specifies the task schedule as a cron expression. It supports an extended syntax including @monthly, @weekly, @daily, @midnight, @hourly, @every X[h|m|s].
+	// +optional
+	Cron *string `json:"cron,omitempty"`
+
+	// timezone specifies the timezone of cron field.
+	// +optional
+	Timezone *string `json:"timezone,omitempty"`
```

Comment:
if this is truly only for `cron`, maybe it should be named `cronTimezone` to avoid implications on `startDate` field?

Reply:
It's also for 'window', which was excluded from this PR. I think naming it that way would kill potential extensibility. And I would refrain from interfering with the upstream api to much in general, it makes the ux worse imo.

Reply:
> It's also for 'window',

ok, with that is makes sense.